src/Joystick/Joystick.cc

```diff
@@ -653,7 +648,10 @@ void Joystick::_handleAxis()
                 }
             }
             uint16_t shortButtons = static_cast<uint16_t>(buttonPressedBits & 0xFFFF);
-            emit manualControl(roll, -pitch, yaw, throttle, shortButtons, _activeVehicle->joystickMode());
+            //Do not emit joystick commands if the aircraft is not flying if _noCommandsNotFlying
+            if(!(_noCommandsNotFlying && (!_activeVehicle->flying() || !_activeVehicle->armed()))) {
```

Comment:
huh, that's annoying: this prevent joystick calibration on the ground
and also it makes it tricky to check we have good joystick input (since you don't see the joystick slider moving)

we might need to block the mavlink message instead so the rest of the joystick logic works (?)

Reply:
barf. ok.